deals/deals.go

```diff
@@ -103,7 +105,7 @@ func (m *Module) Store(ctx context.Context, waddr string, data io.Reader, dcfgs
 				Root: dataCid,
 			},
 			MinBlocksDuration: dur,
-			EpochPrice:        types.NewInt(c.EpochPrice),
+			EpochPrice:        types.NewInt(uint64(math.Ceil(2 * gbSize * float64(c.EpochPrice)))),
```

Comment:
So, turns out that is a better calculation of the `EpochPrice` to propose.
This will be improved over time.

TL;DR of the calculation:
What we receive in the `dcfgs` is the desired `EpochPrice` of the miners *per Gib, per epoch*.
So the calculation for the stored epoch price its quite simple:
*bytes -> transform to Gib -> multiply per Epoch Per Price desired by miner*

Now, there're are other things that should be solved. The real _Piece Size_ might have some Padding. Also, if the stored thing is in CAR format or not, also affect the calculation. The CAR Format is very close to the ending size; but without CAR the ending size is: original size -> create UnixFS DAG -> CAR the DAG -> store that... so a bit harder to estimate.

For the moment, I'm leaving here the double of the "correct" calculation. I'd raise some questions to Lotus guys about some of this challenges.